modules/ROOT/pages/log-points.adoc

```diff
@@ -1,17 +1,137 @@
 = Anypoint Monitoring Log Points
 
-The Anypoint Monitoring log points feature enables you to interactively extract data from running managed APIs. You can specify properties (for example, the host and method in the request header) in your managed API for which to generate logs without having to write code or redeploy your application. This helps you troubleshoot without having to restart the application. 
+== Log Points for Apps
+
+The Log Points feature for apps enables you to log information from connectors in any Mule app in real time. If you are using one of the supported connectors, you can select a log level (debug, info, warn, or error) and generate logs.
+
+=== Prerequisites
+
+* A https://www.mulesoft.com/anypoint-pricing[Titanium subscription] to Anypoint Monitoring and Mule 3.8.5 or later
+* The *Runtime Manager > Manage Settings* permission
+
+=== Supported Connectors
+
+The following Anypoint Connectors are available for use with the Log Points feature.
+
+.Supported Connectors
+|===
+|Supported Anypoint Connectors |Supported for Mule 3.x |Supported for Mule 4.x
+
+|Amazon S3
+|x
+|x
+
+|Amazon SQS
+|x
+|x
+
+|Database
+|x
+|x
+
+|Email
+|x
+|x
+
+|File
+|x
+|x
+
+|FTP
+|x
+|x
+
+|IBM MQ
+|
+|x
+
+|JMS
+|x
+|x
+
+|Microsoft Service Bus
+|x
+|x
+
+|MongoDB
+|x
+|x
+
+|MQ
+|x
+|x
+
+|Object Store
+|x
+|x
+
+|Salesforce
+|x
+|x
+
+|SFTP
+|x
+|x
+
+|Sockets
+|
+|x
+
+|TCP
+|x
+|
+
+|UDP
+|x
+|
+
+|VM Transport
+|x
+|x
+
+|WMQ
+|x
+|
+
+|===
+
+=== Supported Mule Versions
+
+* Mule 3 versions 3.8.5 and later
+* Mule 4 versions 4.1.1 and later
+
+
+=== Enable App Log Points
+
+. Sign into Anypoint Platform and navigate to Anypoint Monitoring.
+. In the menu on the left, click *Log Points*.
+. Select the following, then click *Select*: +
+* *Environment* - The environment in which the app is running.
+* *Resource name* - The name of the app.
+. Select the connectors for which you want to generate logs.
+. Select a log level for each connector.
+. Click *Save & Apply*. +
+A message that your configuration has been applied appears at the top of the page with a *View API Logs* link.
+. Click the *View Logs* link to view the logs for the selected log point. +
+The logs may take a moment to appear.
+
+[NOTE]
+If you selected trace or debug log levels, you should disable the log point monitoring when you are finished, as it can impact your application's performance.
```

Comment:
```suggestion
If you select either the trace or debug log level, disable log point monitoring when you are finished, to avoid slowing your application's performance.
```